File: ABC126A.java
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        // Your code here!
        Scanner sc = new Scanner(System.in);
        int length = sc.nextInt();
        int index = sc.nextInt();
        String str = sc.next();
        String sa = str.substring(0,index-1);
        String sb = str.substring(index-1,index).toLowerCase();
        String scc = str.substring(index);
        System.out.println(sa + sb + scc);
    }
}

File: C/ABC126A.c
#include <stdio.h>
int main(void){
    // Your code here!
    int length, index;
    scanf("%d%d", &length, &index);
    
    // String型はない。char[]で受け取る。
    // char[]は終結文字を格納するので要素数はn+1（か大きい数）にする
    char s[length + 1];
    scanf("%s", s); // 配列なので、&sでなくs
    
    // 小文字にする
    s[index - 1] += 32;
    
    printf("%s\n", s);
    return 0;
}
